lib/src/components/NxTransferList/types.ts

```diff
@@ -39,3 +50,25 @@ export interface Props<T extends string | number = string> extends StatefulProps
   onAvailableItemsFilterChange: NxFilterInputProps['onChange'];
   onSelectedItemsFilterChange: NxFilterInputProps['onChange'];
 }
+
+export const propTypes: ValidationMap<Props<string | number>> = {
+  allItems: PropTypes.arrayOf(PropTypes.shape({
+    id: PropTypes.oneOfType([
+      PropTypes.string.isRequired,
+      PropTypes.number.isRequired
+    ]).isRequired,
+    displayName: PropTypes.string.isRequired
+  })).isRequired as Validator<DataItem<string | number>[]>,
+  selectedItems: PropTypes.instanceOf(Set).isRequired as Validator<Set<string | number>>,
+  availableItemsLabel: PropTypes.node,
+  selectedItemsLabel: PropTypes.node,
+  availableItemsCountFormatter: PropTypes.func as Validator<(n: number) => string>,
+  selectedItemsCountFormatter: PropTypes.func as Validator<(n: number) => string>,
+  showMoveAll: PropTypes.bool,
+  onChange: PropTypes.func as Validator<(newSelected: Set<string | number>) => void>,
```

Comment:
These casts shouldn't be necessary

Reply:
> These casts shouldn't be necessary

Thanks! This was removed in this commit:
0f0c1c1799991ec9d4c9e64eca29b9c86c4061c0

Reply:
> Thanks! This was removed in this commit:

And the other ones above, on `availableItemsCountFormatter` and `selectedItemsCountFormatter`

Reply:
Thanks! I updated the interface for those two to accept `null` so that I'm able to remove those casts:
8df61c7ba4c2ad10415a869de9993da4bfe5b9af